gfx/drivers_context/mali_fbdev_ctx.c

```diff
@@ -137,31 +179,25 @@ All these workarounds should be reverted when and if egl_destroy issues in libma
 */
 static void gfx_ctx_mali_fbdev_destroy(void *data)
 {
-/*   mali_ctx_data_t *mali = (mali_ctx_data_t*)data;
```

Comment:
Either use #if 0 or remove the commented out block entirely. Preferably the latter if it's never going to be used again, otherwise it's just dead code sitting there for no reason.

Reply:
agreed, but the commit is already removing the entire commented out block actually, isn't it?